scripts/lib/CIME/SystemTests/erp.py

```diff
@@ -57,6 +57,10 @@ def _case_two_setup(self):
         self._case.set_value("CONTINUE_RUN", True)
         self._case.set_value("REST_OPTION","never")
 
+        # May need to change EXEROOT since separate_rundirs is False and separate_builds is True
```

Comment:
I thought to so, but it has to be in the setup (IE case_two_setup), so I didn't feel like I had much choice unless we wanted to provide a base-class version of case_two_setup.